### PR TITLE
Update docs running local

### PR DESCRIPTION
**Pull request by poppastring**
First stab at docs for running dotnet-monitor locally on Windows.

### CODE REVIEW

File: documentation/localmachine.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Running on a local machine
+
+`dotnet monitor` can be installed as global tool providing observability, diagnostics artifact collection, and triggering in local development and testing scenarios [(more details here)](./setup.md#net-core-global-tool).
+
+### Local authentication options
+
+If you are using `dotnet monitor` as local development tool on Windows you have the option to use Windows Authentication which requires no further configuration, [additional security details can be reviewed here](./authentication.md#windows-authentication).
```
Comment (wiktork) [inline documentation/localmachine.md:7]: Mention --no-auth?

File: documentation/localmachine.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+/dump 
+```
+
+```http
+/trace
+```
+
+Alternatively, if you are using an API Key, you must [specify it via the Authorization header](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/authentication.md#authenticating-requests), you can accomplish that with a CLI tool like CURL.
+
+Artifacts will be output to the local download folder by default ('%USERPROFILE%\Downloads'), however, an [egress file provider](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/configuration.md#filesystem-egress-provider) can be defined in the settings file.
```
Comment (wiktork) [inline documentation/localmachine.md:47]: I think this is a bit misleading. That just happens to be the default on a particular browser/client on Windows, but it's not something we control.

File: documentation/localmachine.md (hunk 1 above)
Comment (poppastring) [reply documentation/localmachine.md]: @wiktork Updated.

File: documentation/localmachine.md (hunk 2 above)
Comment (poppastring) [reply documentation/localmachine.md]: @wiktork updated. Please review

File: documentation/localmachine.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+/dump 
+```
+
+```http
+/trace
+```
+
+Alternatively, if you are using an API Key, you must [specify it via the Authorization header](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/authentication.md#authenticating-requests), you can accomplish that with a CLI tool like CURL.
+
+Artifacts can be output to specific local directories by configuring [egress file providers](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/configuration.md#filesystem-egress-provider) in the settings file.
```
Comment (wiktork) [inline documentation/localmachine.md:49]: Maybe something like "in addition to downloading artifacts directly over http, artifacts can be output..."

**Review by wiktork — APPROVED** [review]

File: documentation/localmachine.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+### Local configuration
+
+To monitor a specific local process you can use the settings file to define a default process. This ensures `dotnet monitor` automatically collects artifacts and logs based on a process criteria you have identified (e.g. process name, processs id, etc.).
+
+Defining a default process on Windows requires creating a settings file in the user path (`%USERPROFILE%\.dotnet-monitor\settings.json`). In the following example the default process has a process name of __BuggyDemoCode__.
+
+```json
+{
+"$schema": "https://aka.ms/dotnet-monitor-schema",
```
Comment (kelltrick) [inline documentation/localmachine.md:19]: nit: fix indenting
```suggestion
    "$schema": "https://aka.ms/dotnet-monitor-schema",
    "DefaultProcess": {
        "Filters": [{
            "Key": "ProcessName",
            "Value": "BuggyDemoWeb"
        }]
```

File: documentation/localmachine.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+### dotnet monitor collection
+
+To start using `dotnet monitor` run the following command from a Powershell or Command prompt:
+
+```cmd
+dotnet-monitor collect
+```
+
+Assuming your default process is running you can use the endpoints exposed by `dotnet-monitor` to view metrics, logs or download traces and memory dumps. When using Windows Authentication, your browser will automatically handle the Windows authentication challenge and as such you can navigate to this endpoints directly.
```
Comment (jander-msft) [inline documentation/localmachine.md:36]:
```suggestion
Assuming your default process is running you can use the endpoints exposed by `dotnet monitor` to view metrics and logs or to download traces and memory dumps. When using Windows Authentication, your browser will automatically handle the Windows authentication challenge and as such you can navigate to these endpoints directly. 
```

File: documentation/localmachine.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+```json
+{
+    "$schema": "https://aka.ms/dotnet-monitor-schema",
+    "DefaultProcess": {
+        "Filters": [{
+            "Key": "ProcessName",
+            "Value": "BuggyDemoWeb"
+        }]
+}
```
Comment (jander-msft) [inline documentation/localmachine.md:25]: You are missing a closing brace, which is causing the rest of the file to be rendered incorrectly.

**Review by jander-msft — APPROVED** [review]

File: documentation/localmachine.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+Defining a default process on Windows requires creating a settings file in the user path (`%USERPROFILE%\.dotnet-monitor\settings.json`). In the following example the default process has a process name of __BuggyDemoCode__.
+
+```json
+{
+    "$schema": "https://aka.ms/dotnet-monitor-schema",
+    "DefaultProcess": {
+        "Filters": [{
+            "Key": "ProcessName",
+            "Value": "BuggyDemoWeb"
```
Comment (kkeirstead) [inline documentation/localmachine.md:23]: Above, the process name is listed as `BuggyDemoCode`, but here it's specified as `BuggyDemoWeb`

File: documentation/localmachine.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
+# Running on a local machine
+
+`dotnet monitor` can be installed as global tool providing observability, diagnostics artifact collection, and triggering in local development and testing scenarios [(more details here)](./setup.md#net-core-global-tool).
+
+### Local authentication options
+
+If you are using `dotnet monitor` as local development tool on Windows you have the option to use Windows Authentication which requires no additional configuration.
+
+There are a number of local development scenarios that are much more efficient without authentication, `dotnet monitor` can explicitly configured to [disable authentication](./authentication.md#disabling-authentication). Additional [security details can be reviewed here](./authentication.md#windows-authentication).
```
Comment (kkeirstead) [inline documentation/localmachine.md:9]:
```suggestion
There are a number of local development scenarios that are much more efficient without authentication; `dotnet monitor` can be explicitly configured to [disable authentication](./authentication.md#disabling-authentication). Additional [security details can be reviewed here](./authentication.md#windows-authentication).
```

**Review by kelltrick — APPROVED** [review]

**Pull request merged**